### ISSUE TITLE
New tab button has visible top and right border

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#6467]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
New tab button has visible top and right border

**Expected behavior:**
Should not show the border around new tab button

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
```
 
Brave: 0.13.0 
rev: 0c7a64d6917b6e7e72d074b05e1d7488466418a9 
Muon: 2.0.8 
libchromiumcontent: 54.0.2840.100 
V8: 5.4.500.41 
Node.js: 7.0.0 
Update Channel: dev 
os.platform: win32 
os.release: 6.2.9200 
os.arch: x64
```

- Steps to reproduce:
    1. Do a clean install of 0.13.0 Preview 7
    2. New tab button has a visible top border
    3. Open enough tabs to make the newtab button close to the hamburger menu, new tab button has clearly visible top and right border

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/21565013/78f009e2-ce8a-11e6-8439-26ac5d851953.png)

- Any related issues:
cc: @jkup

**Comment by luixxiul** [comment]
Test plan: available on the 1st post.

**Comment by luixxiul** [comment]
I added `release-notes/exclude` as the commits which caused the issue https://github.com/brave/browser-laptop/commit/010e708d4cc2539a0adc5bb3c84b7629865335d7 https://github.com/brave/browser-laptop/commit/615a6ec152a4535531a5cf46b7d6489d73060d62 were included in 0.13.0 (though tagged with 0.12 series)
